tests/acceptance/features/webUIPasswordAddUser/createUserLowercaseLetters.feature

```diff
@@ -0,0 +1,61 @@
+@webUI @mailhog
```

Comment:
The folder name ends with ``AddUser`` - good
To be consistent, the feature files should also have names containing ``addUser``